data/recipes/grr_artifact_turbinia.json

```diff
@@ -0,0 +1,75 @@
+{
+    "name": "grr_artifact_turbinia",
+    "description": [
+        "Collect artifacts from hosts using GRR and processes them with Turbinia.",
+        "",
+        "- Collect a predefined list of artifacts from hosts using GRR",
+        "- Create a tarball of the output and scp it to a remote system",
+        "- Process them with a remote instance of Turbinia"
+    ],
+    "short_description": "Fetches default artifacts from a list of GRR hosts and scp's them to remote system and tells Turbinia to process them.",
+    "modules": [{
+        "wants": [],
+        "name": "GRRArtifactCollector",
+        "args": {
+            "hosts": "@hosts",
+            "reason": "@reason",
+            "grr_server_url": "@grr_server_url",
+            "grr_username": "@grr_username",
+            "grr_password": "@grr_password",
+            "artifacts": "@artifacts",
+            "extra_artifacts": "@extra_artifacts",
+            "use_tsk": "@use_tsk",
+            "approvers": "@approvers",
+            "verify": "@verify"
+        }
+    }, {
+      "wants": ["GRRArtifactCollector"],
+      "name": "LocalFilesystemCopy",
```

Comment:
I *think* you can omit the LocalFilesystemCopy module here, unless you explicitly want the files to be copied to in some user-defined directory.

Reply:
This is still required in this case because the local filesystem module is what I'm using to do the compression.